Documentation/planning/arcade-powered-source-build/README.md

```diff
@@ -0,0 +1,167 @@
+# DRAFT
+
+**This is a proposal, checked in for reference and review. It has not been
+ratified.**
```

Comment:
Out of curiosity, what does "ratification" mean here? Who needs to sign-off?

Reply:
My goal adding this was to make it clear (even just seeing the file content) that repo/infra owners and those involved in .NET 5.0 planning hadn't really looked at this yet. "Ratifying" would be getting a consensus among those people that we're going with this plan as the way to get source-build more sustainable.

I'm going to go ahead and remove this line since we're making progress, and I think we'll only get around to merging this when it's actually the plan. Thanks for calling it out.

(On the flip side of this, the planning information going forward should be on the github issues, and they should point to the doc as necessary. They're fanned out from this ZenHub epic: https://github.com/dotnet/source-build/issues/1510.)